dot/network/host.go

```diff
@@ -159,7 +159,7 @@ func newHost(ctx context.Context, cfg *Config) (*host, error) {
 		return nil, err
 	}
 
-	ps, err := pstoreds.NewPeerstore(ctx, ds, pstoreds.DefaultOpts())
+	ps, err := pstoremem.NewPeerstore(ctx, ds, pstoremem.WithMaxProtocols(1024))
```

Comment:
this is the only option left available from the default options of the deprecated implementation